engines/sci/engine/script_patches.cpp

```diff
@@ -2934,6 +2934,84 @@ static const uint16 qfg3PatchChiefPriority[] = {
 	PATCH_END
 };
 
+// There are 3 points that can't be achieved in the game. They should've been
+// awarded for telling Rakeesh and Kreesha (room 285) about the Simabni
+// initiation.
+// However the array of posibble messages the hero can tell in that room
+// (local 156) is missing the "Tell about Initiation" message (#31) which
+// awards these points.
+// This patch adds the message to that array, thus allowing the hero to tell
+// that message (after completing the initiation) and gain the 3 points.
+// A side effect of increasing the local156 array is that the next local
+// array is shifted and shrinks in size from 4 words to 3. The patch changes
+// the 2 locations in the script that reference that array, to point to the new
+// location ($aa --> $ab). It is safe to shrink the 2nd array to 3 words
+// because only the first element in it is ever used.
```

Comment:
TL;DR - I believed it's safe.

The long answer:

The array is saved to `egoTell.arrays`.
`arrays` is used in 2 places in the base `Teller` class:

      (method (doChild key &tmp index)
         (= index 0)
         (while (++ index)
            (if (== (WordAt keys index) key)
               (self stuffArray: (WordAt arrays index) 1)
               (return 1)
            )
         )
         (return 1)
      )
      
      (method (showDialog)
         (self
            doDialog: (if (== (WordAt arrays 0) curArray) 1 else 0) &rest
         )
      )

`doChild` is overridden in the egoTell instance, so it is never called.
`showDialog` is called, but it only reads `arrays[0]`, so there's no harm in decreasing it to 3 words.